.rubocop_rails.yml

```diff
@@ -253,3 +253,5 @@ Rails/WhereExists: # new in 2.7
   Enabled: true
 Rails/WhereNot: # new in 2.8
   Enabled: true
+Rails/SkipsModelValidations:
```

Comment:
This seems useful to have enabled for production code, is there any way to disable it only for test code?

Reply:
It’s tough… validations are good but I don’t actually care to spend the time to run them while using `touch`, only while creating or updating other fields. We could turn the lint back on and only allow `touch` if you want.

Reply:
Oh that would be a nice compromise. But yeah in my ideal world as many of our validations would be enforced by the database layer anyway (see #20 as an example 😬)